packages/admin-next/dashboard/src/components/layout/shell/shell.tsx

```diff
@@ -200,20 +203,20 @@ const ThemeToggle = () => {
 }
 
 const Logout = () => {
-  // const navigate = useNavigate()
-  // const { mutateAsync: logoutMutation } = useAdminDeleteSession()
+  const navigate = useNavigate()
+  const { mutateAsync: logoutMutation } = useLogout()
 
   const handleLayout = async () => {
-    // await logoutMutation(undefined, {
-    //   onSuccess: () => {
-    //     /**
-    //      * When the user logs out, we want to clear the query cache
-    //      */
-    //     queryClient.clear()
-    //     navigate("/login")
-    //   },
-    // })
-    // noop
+    await logoutMutation(undefined, {
+      onSuccess: () => {
+        /**
+         * When the user logs out, we want to clear the query cache
+         */
+        queryClient.clear()
+        console.log("Hello world")
```

Comment:
Do we want to log hello world here? 😆

Reply:
Haha, I thought that would be a nice easter egg 😆

Reply:
Maybe we can log our recruitment description ?? ☺️